pkg/operator/ceph/controller/controller_utils.go

```diff
@@ -81,6 +82,16 @@ func DiscoveryDaemonEnabled(context *clusterd.Context) bool {
 	return value == "true"
 }
 
+// CephCommandsTimeout is the timeout value of Ceph commands which are executed from Rook
+func CephCommandsTimeout(context *clusterd.Context) time.Duration {
+	strTimeout, _ := k8sutil.GetOperatorSetting(context.Clientset, OperatorSettingConfigMapName, "ROOK_CEPH_COMMANDS_TIMEOUT", "15")
```

Comment:
Every time this is called, the operator settings configmap is loaded, so let's avoid calling it so often for every command. But with the suggestion in my previous comment, this means the the operator only needs to call `GetOperatorSetting()` once, then set the `exec.CephCommandTimeout` once.